revert_deploy.yml

```diff
@@ -0,0 +1,55 @@
+###
+#  Set to previous deploy in case of an error or problem.
+#
+# Example usage:
+# ansible-playbook -e host_group=prosody_qa revert_deployment.yml
+#
+# The script will ask for confirmation before making any breaking changes.
+# ctrl+c and then 'a 'will break execution, enter or ctrl+c and then 'c' will
+# proceed.
+#
+# NOTE: You will need the password file loaded for any secrets encrypted in
+# the vaults used by the host_group specified.
+###
+
+- hosts: '{{ host_group }}'
+  connection: ssh
+  remote_user: deploy
+  environment:
+    PATH: '{{ path }}'
+    LD_LIBRARY_PATH: '{{ ld_library_path }}'
+    PYTHONPATH: '{{ python_path }}'
+  tasks:
+  - name: Pause and let user make sure command looks right.
+    pause:
+      prompt: >
+               You are about to revert {{ host_group | upper }}.
+               Press enter to proceed or ctrl+c then a to abort.
+  - name: Check that previous symlink exists.
+    stat:
+      path: "{{ install_root }}/previous"
+    register: previous
+  - fail:
+      msg: "Previous symlink does not exist."
+    when: previous.stat.islnk is not defined
+  - name: Get current symlink path
+    stat:
+      path: "{{ install_root }}/current"
+    register: current
+  - fail:
+      msg: "Current symlink does not exist."
```

Comment:
Does it matter if there is no `current` ? I can't imagine a scenario where we'd have a previous and no current, but it seem that from the perspective of reverting a deploy we wouldn't care whether or not there is a current deploy.